docs/docusaurus/docs/reference/learn/data_quality_use_cases/integrity.md

```diff
@@ -0,0 +1,278 @@
+* **Value dependencies**: Hierarchies and dependencies in numerical and categorical data must be maintained. For example, a project management system should ensure that task start dates precede end dates and that parent tasks span the duration of their subtasks.
+
+Failing to validate these relationships can result in data discrepancies, incorrect calculations, and flawed insights, undermining the reliability and value of your data assets.
+
+Great Expectations (GX) offers a robust framework for validating data integrity at scale. Its expressive Expectations, flexible validation workflows, and support for diverse data sources enable data teams to implement comprehensive integrity checks across their entire data ecosystem.
+
+In this guide, we'll demonstrate how to use GX to define and enforce data integrity rules through practical examples and best practices. We'll cover techniques for validating referential integrity, cross-table consistency, and compliance with business rules.
+
+## Prerequisite knowledge
+
+This article assumes basic familiarity with GX components and workflows. If you're new to GX, start with the [GX Cloud](https://docs.greatexpectations.io/docs/cloud/overview/gx_cloud_overview) and [GX Core](https://docs.greatexpectations.io/docs/guides/overview) overviews to familiarize yourself with key concepts and setup procedures.
+
+## Data preview
+
+The examples in this guide use a sample dataset of financial transactions and transfer balances, provided from public Postgres database tables. The sample data is also available in CSV format for each table:
+
+
+1. `transactions` table ([CSV](https://raw.githubusercontent.com/great-expectations/great_expectations/develop/tests/test_sets/learn_data_quality_use_cases/integrity_transactions.csv)):
+
+```
+| transaction_id | sender_id | recipient_id | amount  | transaction_date | reference_number | confirmation_code |
+|----------------|-----------|--------------|---------|------------------|------------------|-------------------|
+| 1001           | 501       | 502          | 250.00  | 2024-01-15       | 1001             | 1001              |
+| 1002           | 502       | 503          | 40.00   | 2024-01-15       | 1002             | 1002              |
+| 1003           | 503       | 501          | 1200.00 | 2024-01-16       | 1003             | 1003              |
+| 1004           | 504       | 502          | 80.00   | 2024-01-16       | 1004             | 1004              |
+```
+
+2. `transfer_balances` table ([CSV](https://raw.githubusercontent.com/great-expectations/great_expectations/develop/tests/test_sets/learn_data_quality_use_cases/integrity_transfer_balances.csv)):
+
+```
+| transaction_id | transfer_amount | adjustment | sender_debit | recipient_credit |
```

Comment:
I think it would be helpful to distinguish between the two datasets by having different pk columns - is there any reason we wouldn't want this column to be `transfer_id`?